docs/deployment/deployment-checklist.md

```diff
@@ -111,9 +111,10 @@ The following checklist breaks down permissions required by Semgrep features.
 <Tabs
     defaultValue="gh"
     values={[
-    {label: 'GitHub', value: 'gh'},
```

Comment:
Silly question from me - should we group by popularity or by alphabetization? I'm sure at least 50% of the time people will need to click the GH tab 😓 They are the most popular.

Reply:
I alphabetized this due to my neurosis, but I left GH as the default, so it should be showing most of the time anyway; if you think it should be by popularity, I don't mind changing it back!

Reply:
@khorne3 I didn't notice, thank you - yeah GH is the default, I should have tested first.